source/creating_maps_charts/02_creating_advanced_charts/analyze.rst

```diff
@@ -48,91 +47,114 @@ Here is what this layer looks like with these settings and two thresholds:
+Try changing the type from **Line** to **Spline** to see a nice curve on top of the initial column chart.
 
-In the next section we will add another dataset to our chart.
+You can reorder the series by clicking and dragging. The series order affects the chart legend, and the series at the top has priority on the **Y-Axis** parameter.
 
 
 Stacking layers
 ---------------
 
-Let's elaborate on another data visualization, this time with multiple layers.
+You create data visualizations with multiple layers. Such visualizations can help you find trends in data.
 
-**First Layer**
 
-We will start the same way as before, click on ``Add a dataset to this layer``, then search for "Oil Prices" to add the `Oil Prices <https://public-us.opendatasoft.com/explore/dataset/oil-prices>`_ dataset.
+Creating a chart with multiple layers
+-------------------------------------
 
-No need to filter it, just note that the data starts from the year 1987. We will use 1987 as the starting point for the other layers.
+For example, you can create a data visualization to find whether oil prices are correlated with gold prices.
 
-When you first import a dataset in Advanced Charts, the default visualization of this dataset is imported. In this case, you can see two lines: `WTI Spot Price` and `Brent Spot Price`.
+This example of chart relies on the following datasets:
 
-**Second Layer**
+ - `Oil Prices <https://documentation-resources.opendatasoft.com/explore/dataset/oil-prices/>`__ that contains monthly oil prices from 1988 to 2019, and
+ - `Gold Prices <https://documentation-resources.opendatasoft.com/explore/dataset/gold-prices/>`__ that contains monthly gold prices from 1988 to 2019.
 
-Let's add another dataset! Look for "Commodity Prices" to find `Commodity Prices since 1980 <https://public.opendatasoft.com/explore/dataset/commodity-prices-since-1980>`_ dataset from the Opendatasoft public collection.
+This chart compares Brent and WTI prices with gold prices between 1989 and 2019.
 
-This time, add the filter "`date>1987`" so that the X-axis will match with the first layer. Then import the ~20 000 records.
+To follow this example, add these datasets as sources.
 
-The default visualization is a line with the average commodity price.
+Step 1: Add a first layer
+^^^^^^^^^^^^^^^^^^^^^^^^^
 
-You should see your query filter just under the dataset name: "`query: date>1987`".
+Add the "Oil Prices" dataset as a first layer on this chart.
 
-.. admonition:: Note
-   :class: note
+1. Access the Chart Builder.
 
-   You can click on the pencil icon |edit-pencil| of a layer to change the selected dataset or refine the query.
+2. Click on the **Add a dataset to this layer** button.
 
-.. |edit-pencil| image:: images/edit.png
+3. Search for "Oil Prices" and select the "Oil Prices" dataset.
+
+A line has been added to the chart.
```

Comment:
I edited the default visualization so that users must go though step 2. The idea is that they explore the "adding a series" feature.
I re-tested after this update [here](https://documentation-resources.opendatasoft.com/chart/?dataChart=eyJ0aW1lc2NhbGUiOiJtb250aCIsInF1ZXJpZXMiOlt7ImNoYXJ0cyI6W3sidHlwZSI6ImxpbmUiLCJmdW5jIjoiQVZHIiwieUF4aXMiOiJ3dGlfcHJpY2UiLCJzY2llbnRpZmljRGlzcGxheSI6dHJ1ZSwiY29sb3IiOiIjNjZjMmE1In1dLCJ4QXhpcyI6ImRhdGUiLCJtYXhwb2ludHMiOiIiLCJ0aW1lc2NhbGUiOiJtb250aCIsInNvcnQiOiIiLCJjb25maWciOnsiZGF0YXNldCI6Im9pbC1wcmljZXMiLCJvcHRpb25zIjp7InRpbWV6b25lIjoiRXVyb3BlL0JlcmxpbiJ9fX1dLCJhbGlnbk1vbnRoIjp0cnVlLCJkaXNwbGF5TGVnZW5kIjp0cnVlfQ%3D%3D), and there is just one line that appears now.